clients/ts/signalr/tests/WebSocketTransport.test.ts

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+import { ILogger } from "../src/ILogger";
+import { TransferFormat } from "../src/ITransport";
+import { NullLogger } from "../src/Loggers";
+import { WebSocketTransport } from "../src/WebSocketTransport";
+import { VerifyLogger } from "./Common";
+import { TestMessageEvent } from "./TestEventSource";
+import { TestCloseEvent, TestEvent, TestWebSocket } from "./TestWebSocket";
+
+describe("WebSocketTransport", () => {
+    [[TransferFormat.Text, "blob"],
+    [TransferFormat.Binary, "arraybuffer"]]
+        .forEach(([input, expected]) => {
+            it(`sets websocket binarytype based on transferformat: ${TransferFormat[input]}`, async () => {
+                await VerifyLogger.run(async (logger) => {
+                    const webSocket = new WebSocketTransport(undefined, logger, true, TestWebSocket);
+
+                    const connectPromise = webSocket.connect("http://example.com", input as TransferFormat);
+
+                    await TestWebSocket.webSocket.openSet;
+                    TestWebSocket.webSocket.onopen(new TestEvent());
+
+                    await connectPromise;
+
+                    expect(TestWebSocket.webSocket.binaryType).toEqual(expected);
```

Comment:
When checking primitives, we should use `.toBe`. It's equivalent to `===`. I forgot to note that on the SSE version...

Reply:
Just do a scan through this and the SSE file for `.toEqual` and replace it with `.toBe` where it looks appropriate (primitive values)